src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Cropping/CroppingInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Media\ImageConverter\Cropping;
+
+use Imagine\Image\ImageInterface;
+
+/**
+ * Defines a cropping on an image.
+ */
+interface CroppingInterface
```

Comment:
Because these are verbs, and sound like method names to me. Whereby `Cropper` could be considered a noun, which translates to a class for me.